<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fereastrastergeDepozit</class>
 <widget class="QDialog" name="fereastrastergeDepozit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>799</width>
    <height>499</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#fereastrastergeDepozit{
background:url(:/imaginebuna.png)
}</string>
  </property>
  <widget class="QLabel" name="labelInformatie">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>400</y>
     <width>221</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>350</y>
     <width>113</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>320</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Introdu ID Depozit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>440</y>
     <width>83</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>STERGE</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>10</y>
     <width>83</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Inapoi</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>50</y>
     <width>401</width>
     <height>121</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>200</y>
     <width>83</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Afiseaza</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
